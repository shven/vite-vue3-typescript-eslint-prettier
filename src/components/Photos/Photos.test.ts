import { mount } from '@vue/test-utils';
import Component from './VuePhotos.vue';

describe('VuePhotos.vue', () => {
    const wrapper = mount(Component);

    it('Match snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
