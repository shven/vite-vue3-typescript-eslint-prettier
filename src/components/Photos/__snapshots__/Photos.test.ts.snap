// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`VuePhotos.vue > Match snapshot 1`] = `
<div
  class="photos"
  data-v-afbfae38=""
>
  <div
    class="container"
    data-v-afbfae38=""
    data-v-cbb0d1af=""
  >
    
    <h1
      class="title title--large"
      data-v-a0ca2fca=""
      data-v-afbfae38=""
    >
      Photos
    </h1>
    <div
      data-v-afbfae38=""
    >
      Loading...
    </div>
    
  </div>
</div>
`;
